__main__.py

```diff
@@ -3,81 +3,154 @@
 import datetime as dt
 import json
 import sys
+from enum import Enum, auto
 from pathlib import Path
-from typing import NoReturn
+from typing import TYPE_CHECKING, Any, ClassVar, NewType, NoReturn, TypedDict, cast
 
 import requests
 from dahlia import dahlia, dprint
 
+if TYPE_CHECKING:
+    from typing_extensions import Self
+
+
+__version__ = "0.1.0"
+
+
+class Status(str, Enum):
+    def _generate_next_value_(  # type: ignore
+        name: str, start: int, count: int, last_values: list[str]
+    ) -> str:
+        # https://github.com/python/mypy/issues/7591
+        return name.title().replace("_", " ")
```

Comment:
Why `.replace("_", " ")`? I don't see no `_` in `Status` members.
The enum itself rather feels unlikely to change.

Reply:
IIRC I added this to future-proof it if there ever happens to be an item like `FOO_BAR = "Foo Bar"`. But I agree it sounds unlikely, and if it ever happens, this could be re-added.

Reply:
```suggestion
        return name.title()
```